src/sources/mongodb_metrics/mod.rs

```diff
@@ -1118,7 +1122,7 @@ mod integration_tests {
             tokio::spawn(async move {
                 MongoDbMetricsConfig {
                     endpoints,
-                    scrape_interval_secs: 15,
+                    scrape_interval_secs: Duration::from_secs(15),
```

Comment:
👀

Reply:
I enabled integration tests for this PR... better safe than sorry :)

Reply:
😆

Reply:
burned too many times :ded: